tests/js/server/aql/aql-modify-cluster.js

```diff
@@ -69,11 +71,19 @@ function ahuacatlModifySuite () {
     setUp : function () {
       db._drop(cn1);
       db._drop(cn2);
+      db._drop(cn3);
       c1 = db._create(cn1, {numberOfShards:5});
       c2 = db._create(cn2, {numberOfShards:5});
+      c3 = db._create(cn3, {numberOfShards:5});
```

Comment:
We are already creating 2 new collections for each test case.
Why do we need a 3rd one that is unused in most test cases, and used only in a few new ones?
Can't we just use `c1` or `c2` in the new test cases?

Reply:
Removed c3